### DISCUSSION TITLE
Server crashe after saving.

### DISCUSSION THREAD

**Discussion by Katzenkralle** [Pryaxis/TShock#1862] [Q&A]
Hello,
I have the problem that the server crashes after some autosaves. I use the Raspberry Pi3 B + with mono and Tshock version 4.4.0.
Thank you for your help.

**Comment by Katzenkralle** [comment]
Sry, I use version 1.4.0.3.

**Reply by matthewstaylor** [reply]
Same problem on my end. Server saves world to 100% and stop responding.

**Reply by matthewstaylor** [reply]
Confirmed this issue with Raspberry Pi 3 with pre-release 6 even after fresh install. World data gets saved to 100% and my rasbpi stops responding. If I close my SSH connection I cannot attempt to reconnect at all either. Last stable version ran great on same rasbpi build.

Edit: Raspberry Pi just cant hang with large worlds. Works with a medium world fine.
